labcodes/labstorybook
# Add Toggle component

This PR will add the Toggle React component, its styles and the Storybook story.
Also adds JS tests setups, and the Toggle component tests.

**How to test:**
- Run the project and check if the Toggle styles are correct (compare with Figma components)
- Remember to check if it works fine on hover, on focus and when value changes
- Check if any functionality is missing
- From the project root, run `cd labsystem/ && yarn test`, check if they all passes

**Screenshots:**
![image](https://user-images.githubusercontent.com/29437811/81997974-b30bb000-9627-11ea-8493-2e6be600aa89.png) 
![image](https://user-images.githubusercontent.com/29437811/81997989-bdc64500-9627-11ea-98db-e75832545053.png)
![image](https://user-images.githubusercontent.com/29437811/81998006-c9b20700-9627-11ea-8e5b-3799ac55b0b4.png)
